### PR TITLE
Issue 77 - Roster Endpoint

### PR DESCRIPTION
**Pull request by calebedwards21**
Add the following endpoint "**/rosters**":
- Get 
  - Get all rosters
- Post
  - Create a roster given an email for a captain and a list of emails for players

Added openapi docs 

TODO: Add tests

### CODE REVIEW

File: static/oas/v1/rosters/rosters.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      tags: 
+        - roster
+      summary: Create Roster  
+      requestBody:
+        description: A JSON object with a valid roster
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/schemas/PostRosterRequest"
```
Comment (tannerwheeler) [inline static/oas/v1/rosters/rosters.yml]: I believe everywhere where you see #/schemas... you need to change it to #/components/schemas...
This is what I am seeing in the current code.

File: static/oas/v1/rosters/rosters.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          application/json:
+            schema:
+              $ref: "#/schemas/PostRosterRequest"  
+      responses: 
+        200:
+          description: Successfully create a roster by captain
+          content: 
+            application/json: 
+              schema: 
+                $ref: '#/schemas/PostRosterResponse'
```
Comment (tannerwheeler) [inline static/oas/v1/rosters/rosters.yml]: I believe everywhere where you see #/schemas... you need to change it to #/components/schemas...
This is what I am seeing in the current code.

File: static/oas/v1/rosters/rosters.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+      tags:
+        - roster
+      summary: All rosters
+      responses:
+        200:
+          description: Successfully get all rosters
+          content:
+            application/json:
+              schema:
+                $ref: '#/schemas/GetRostersResponse'
```
Comment (tannerwheeler) [inline static/oas/v1/rosters/rosters.yml]: I believe everywhere where you see #/schemas... you need to change it to #/components/schemas...
This is what I am seeing in the current code.

**Review by tannerwheeler — CHANGES_REQUESTED** [review]
Everything looks as far as I am aware except in the rosters.yml. There was a recent change in the code I think affects RapiDocs

File: static/oas/v1/rosters/rosters.yml (hunk 1 above)
Comment (calebedwards21) [reply static/oas/v1/rosters/rosters.yml]: changed

File: static/oas/v1/rosters/rosters.yml (hunk 2 above)
Comment (calebedwards21) [reply static/oas/v1/rosters/rosters.yml]: changed

File: static/oas/v1/rosters/rosters.yml (hunk 3 above)
Comment (calebedwards21) [reply static/oas/v1/rosters/rosters.yml]: changed

**Review by tannerwheeler — APPROVED** [review]

**Pull request merged**